core/config/src/main/resources/default.conf

```diff
@@ -55,6 +55,9 @@ dataset {
   # the file size limit for dataset upload
   single_file_upload_max_size_mb = 20
 
+  # the maximum number of files uploading in parallel
```

Comment:
You don't need comment for this. The variable name and the comment is pretty much the same.